controllers/userProfileAPI.js

```diff
@@ -0,0 +1,5 @@
+const express = require('express');
+const userProfile = require('../models/userProfile')
+const userProfileView = require('../views/userProfileView');
+const router = express.Router();
+const jsonResponse = require('../utils/JSONResponse');
```

Comment:
I personally reckon we should put the these api calls in the userAPI.js class,

and just have like '/profile' as an api call, which will effectively be 'localhost:3000/user/profile'

Reply:
Yeah that makes more sense